homeassistant/components/plex/update.py

```diff
@@ -73,6 +71,18 @@ def release_notes(self) -> str | None:
     def install(self, version: str | None, backup: bool, **kwargs: Any) -> None:
         """Install an update."""
         try:
-            self.plex_server.installUpdate()
+            self._server.plex_server.installUpdate()
         except (requests.exceptions.RequestException, PlexApiException) as exc:
             raise HomeAssistantError(str(exc)) from exc
+
+    @property
+    def device_info(self) -> DeviceInfo | None:
+        """Return a device description for device registry."""
+        return DeviceInfo(
```

Comment:
can we set this in `self._attr_device_info`?

Reply:
I reused the logic from the other entities from the plex integration. It may use a old way to do it.